plugin/trino-postgresql/src/test/resources/postgresql.conf

```diff
@@ -0,0 +1,3 @@
+listen_addresses = '*'
+log_destination = 'stderr'
```

Comment:
will this cause more output from CI jobs?

Reply:
No. We don't read output from containers.